app/builders/collection.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-module ToCollectionForm
-  # Maps Cocina Collection to CollectionForm
-  class Mapper
+module Builders
```

Comment:
`Builders` doesn't make sense to me, since it is a mapping from one specific thing to another. My expectation is that a builder helps you build things (like XML documents).

Also, that this is mapping to a form is lost in this naming.

Reply:
I was reluctant to use `FormBuilders` because those are already a Rails built-in thing. I do like the idea of conveying forminess with this name since that's the output. I was partial to builders only because we're building up the form object as we pick out the bits of Cocina that correspond to our forms, but I don't love the name. (I'm not wedded to any of the names in this PR. In fact, if we're actively discussing what's the best name for any of these, the PR is already a success.) Form factories? Form mappers? 

Whatever choices we make, if we decide to take this stuff forward, I'm committing to documenting in the README so that we can make sense of this in the months and years to come.